Add Development Guide for Main Feast Repo Feast Components (#1424)
feast-dev/feast: README.md

 * [Change Log](https://github.com/feast-dev/feast/blob/master/CHANGELOG.md)
 * [Slack (#Feast)](https://join.slack.com/t/tectonfeast/shared_invite/zt-n7pl8gnb-H7dLlH9yQsgbchOp36ZUxQ)

## Contributing

> Feast is a community project and is still under active development.Your feedback and contributions are important to us. Please have a look at our contributing and development guides:

Reviewer: ```suggestion
Feast is a community project and is still under active development. Please have a look at our contributing and development guides if you want to contribute to the project:
```
